data/provider/src/test/java/tech/pegasys/artemis/provider/JsonProviderTest.java

```diff
@@ -28,7 +28,7 @@
   void printBeaconStateTest() {
```

Comment:
maybe we should rename the test now.  Something like `BeaconStateToJsonTest`?

Reply:
Makes sense.